### PR TITLE
remove dead code and unused imports

### PR DESCRIPTION
**Pull request by butonic**
Two services reference classes that no longer exist ... I ran a quick inspection and removed other unused imports in the dav app.

@DeepDiver1975

### CODE REVIEW

**Comment by PVince81** [comment]
:+1: but I'd like @DeepDiver1975 to confirm the service removals regarding birthday

File: apps/dav/lib/AppInfo/Application.php [hunk 1]
```diff
@@ -111,23 +106,6 @@ public function __construct (array $urlParams=array()) {
 			);
 		});
 
-		$container->registerService('OCA\DAV\Migration\Classification', function ($c) {
```
Comment (DeepDiver1975) [inline apps/dav/lib/AppInfo/Application.php]: :+1:

File: apps/dav/lib/AppInfo/Application.php [hunk 2]
```diff
@@ -111,23 +106,6 @@ public function __construct (array $urlParams=array()) {
...
-			/** @var IAppContainer $c */
-			return new Classification(
-				$c->query('CalDavBackend'),
-				$c->getServer()->getUserManager()
-			);
-		});
-
-		$container->registerService('OCA\DAV\Migration\GenerateBirthdays', function ($c) {
```
Comment (DeepDiver1975) [inline apps/dav/lib/AppInfo/Application.php]: :+1:

**Comment by DeepDiver1975** [comment]
:+1:

**Pull request merged**